packages/cli/README.md

```diff
@@ -101,6 +167,119 @@ To transform your project stylesheets to target JavaScript modules containing th
 $ stc --srcDir="./src" --outDir="./dist"
 ```
 
+## Multiple Projects
+
+Projects allow you to share `stc` configuration and manage your Stylable projects in one place with predictable and controlled build order with caching optimizations.
+
+```ts
+export interface MultipleProjectsConfig extends Partial<SingleProjectConfig> {
+    presets?: Presets;
+    projects: Projects;
+    projectsOptions?: {
+        resolveRequests?: ResolveRequests;
+    };
+}
+```
+
+> Example for simple monorepo with Stylable packages
+```js
+const { typedConfiguration } = require('@stylable/cli');
+
+exports.stcConfig = typedConfiguration({
+    options: {
+        srcDir: './src',
+        outDir: './dist',
+        outputSources: true,
+        cjs: false,
+        useNamespaceReference: true, 
+    },
+    projects: ['packages/*']
+});
+
+```
+
+### Options
+
+Like for [single project](#configuration-file) `options` is the top-level `BuildOptions` and is the default options for each project.
+
+### Projects
+
+**Projects** is a generic term that refers to the set of requests with defined single or multiple `BuildOptions`.\
+This set of requests is being processed and then evaluated as a map of `projectRoot` (directory path) to a set of `BuildOptions`.
+
+By default, the request is a path to a package, and in order to make the correct topological sort, the dependency needs to be specified in each package `package.json`
+
+As said, the value of the request can be resolved to a single `BuildOptions`, but it can also be multiple builds.
+
+```jsonc
+{
+    //...
+    projects: {
+        "packages/*": { 
+            // ...BuildOptions
+        },
+        "other-package/*": [
+            { /* #1 ...BuildOptions */ }, 
+            { /* #2 ...BuildOptions */ }, 
+        ]
+    }
+    //...
+}
+```
+
+> The full types specification for defining Projects
+```ts
+export type Projects =
+    | Array<string | [string, ProjectEntryValues]>
+    | Record<string, ProjectEntryValues>;
+
+// P = Preset name
+export type ProjectEntryValue<P extends string> =
+    | P
+    | PartialBuildOptions
+    | {
+          preset?: P;
+          presets?: Array<P>;
+          options: PartialBuildOptions;
+      };
+
+export type ProjectEntryValues<P extends string> =
```

Comment:
maybe switch `ProjectEntryValues` to be above `ProjectEntryValue` as it is used first and then uses the other.